flink-connectors/flink-connector-hive/src/test/java/org/apache/flink/connectors/hive/HiveDialectITCase.java

```diff
@@ -467,14 +468,36 @@ public void testAddDropPartitions() throws Exception {
 
 	@Test
 	public void testShowPartitions() throws Exception {
-		tableEnv.executeSql("create table tbl (x int,y binary) partitioned by (dt date, country string)");
-		tableEnv.executeSql("alter table tbl add partition (dt='2020-04-30',country='china') partition (dt='2020-04-30',country='us')");
+		tableEnv.executeSql("create table tbl (x int,y binary) partitioned by (country string)");
```

Comment:
Why do we need a different table schema? I think you can just reuse the table in this test